README.md

```diff
@@ -67,8 +67,8 @@ blocks = Slack::BlockKit.blocks do |b|
   b.append(a_prebuilt_block)
 end
 
+body = { blocks: blocks.as_json }
```

Comment:
I've dropped `text: 'New block message!'` because Slack Block Kit Builder UI throws an error when the line is present.